corehq/apps/userreports/pillow.py

```diff
@@ -57,9 +57,18 @@ def rebuild_table(self, table):
         table.rebuild_table()
 
     def python_filter(self, doc):
-        # filtering is done manually per indicator set change_transport
+        # filtering is done manually per indicator see change_transport
         return True
 
+    def change_trigger(self, changes_dict):
```

Comment:
Cory is correct, the second case is what I was seeing in the table, but I promptly rebuilt the table and those cases were not re-inserted. I'll let you know if I see and other cases of this occurring.